front-end/src/components/Card/Card.js

```diff
@@ -1,23 +1,90 @@
-import React from 'react';
```

Comment:
Organize imports: (Libraries, Redux Files, App components, UI components, Styled Components, Utils, Images)

Reply:
The import order has been fixed.